Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -125,6 +125,13 @@ function processAttribute(
     variableName = VertexAttributeSemantic.getVariableName(semantic, setIndex);
     varyingName = "v_" + variableName;
 
+    if (
+      semantic === VertexAttributeSemantic.FEATURE_ID &&
+      setIndex >= renderResources.featureIdVertexAttributeSetIndex
+    ) {
+      renderResources.featureIdVertexAttributeSetIndex = attribute.setIndex + 1;
```

Comment:
The intention is to avoid collision with the existing `FEATURE_ID` attributes. We don't want to create `a_featureId_0` for an attribute that uses `constant`/`divisor`, if there is another feature ID attribute that uses an existing attribute called `FEATURE_ID_0`.